packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -28,6 +29,7 @@ export interface IComponentPageProps {
   otherSections?: IComponentPageSection[];
   allowNativeProps?: boolean | string;
   nativePropsElement?: string | string[] | undefined;
+  feedback?: JSX.Element;
```

Comment:
please jsdoc all of these props (actually #5542 is doing so, so minimally JSdoc feedback)

Reply:
Should this just be a boolean?

Reply:
@cliffkoh I think the file she jsdoc-ed is different so I will jsdoc all the ones in this file